internal/moreos/proc_darwin.go

```diff
@@ -42,3 +44,18 @@ func ensureProcessDone(p *os.Process) error {
 func isExecutable(f os.FileInfo) bool {
 	return f.Mode()&0111 != 0
 }
+
+func cmdStart(cmd *exec.Cmd, cancel context.CancelFunc, writeProcHint func()) error {
```

Comment:
would this be a couple less lines vs declaring a var?

```suggestion
func cmdStart(cmd *exec.Cmd, cancel context.CancelFunc, writeProcHint func()) (err error) {
```